monkey/tests/unit_tests/infection_monkey/network/test_ssh_fingerprinter.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+from infection_monkey.i_puppet import FingerprintData, PortScanData, PortStatus
+from infection_monkey.network.ssh_fingerprinter import SSHFingerprinter
+
+
+@pytest.fixture
+def ssh_fingerprinter():
+    return SSHFingerprinter()
+
+
+def test_no_ssh_ports_open(ssh_fingerprinter):
+    port_scan_data = {
+        80: PortScanData(80, PortStatus.CLOSED, "", "tcp-80"),
```

Comment:
Set the ssh port and it status to closed. This doesn't mean that there are no ssh ports open. It means that they are not configured.